roles/common/defaults/main.yml

```diff
@@ -296,6 +298,7 @@ gi_patches:
 # Warning: GIRU 32126842 for 19.10.0.0.210119 has critical regressions and password protected. Development recommends 19.11.0.0.210420 Patch 32545008 instead of the JAN 2021 quarter RU
 #  - { category: "RU", base: "19.3.0.0.0", release: "19.10.0.0.210119", patchnum: "32126842", patchfile: "p32126842_190000_Linux-x86-64.zip", patch_subdir: "/32226239", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE, md5sum: "opnaS/8dS13yGypZCCN8WA==" }
 #  - { category: "RU", base: "19.3.0.0.0", release: "19.11.0.0.210420", patchnum: "32578973", patchfile: "p32578973_190000_Linux-x86-64.zip", patch_subdir: "/32545008", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE, md5sum: "YLEOruyjCOdDvUOMBUazNQ==" }
+  - { category: "RU", base: "19.3.0.0.0", release: "19.9.0.0.211019", patchnum: "33248471", patchfile: "p33248471_190000_Linux-x86-64.zip", patch_subdir: "/33192694", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE, md5sum: "3LgQa6fZ/UHs3XZ8GvtwrQ==" }
```

Comment:
Verification of patch directory shows: 

```bash
.
├── 33248471
│   ├── 33182768   <== this is the GI RU, some molecules go into only GI_HOME, some into only RDBMS_HOME & some into both
│   │   ├── 32585572  <== goes into only GI_HOME
│   │   ├── 33192793  <== goes into only DB_HOME for non-RAC, both homes for RAC
│   │   ├── 33208107  <== goes only into GI_HOME
│   │   ├── 33208123  <== goes into both homes
│   │   ├── 33239955  <== goes only into GI_HOME
│   │   ├── automation
│   │   ├── bundle.xml
│   │   ├── README.html
│   │   └── README.txt
│   ├── 33192694   <============ this is the OJVM RU, that goes only into the RDBMS home
│   │   ├── etc
│   │   ├── files
│   │   ├── README.html
│   │   └── README.txt
│   ├── PatchSearch.xml
│   └── README.html
└── PatchSearch.xml
```

So, the `patch_subdir: "/33192694"` under `gi_patches:` should actually be `patch_subdir: "/33182768"`, like so:
```
gi_patches:
...
  - { category: "RU", base: "19.3.0.0.0", release: "19.13.0.0.211019", patchnum: "33248471", patchfile: "p33248471_190000_Linux-x86-64.zip", patch_subdir: "/33182768", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE, md5sum: "3LgQa6fZ/UHs3XZ8GvtwrQ==" }
```

And for `rdbms_patches:`, it should not be: `patch_subdir: "/33182768/33192793"`, instead, it should read: `patch_subdir: "/33192694"`, like so:
```
  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.13.0.0.211019", patchnum: "33248471", patchfile: "p33248471_190000_Linux-x86-64.zip", patch_subdir: "/33192694", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE, md5sum: "3LgQa6fZ/UHs3XZ8GvtwrQ==" }

```

And also, a typo:
Change `release: "19.9.0.0.211019"` to `release: "19.13.0.0.211019"` under both `rdbms_patches:` and `gi_patches:` data structures.

Reply:
I have submitted the corrected version for 19c metadata in [pull 104](https://github.com/google/bms-toolkit/pull/104).

You may discard the 19c changes patch metadata here and proceed to correct the 12c / 18c patch metadata in this PR.

Following may be an useful reference:
https://github.com/google/bms-toolkit/blob/master/docs/patch-metadata.md